driver/pelican/interface.md

```diff
@@ -35,6 +37,30 @@ Next, it's wise if we attempt to define what a "daily schedule" actually looks l
 
 Going one layer above, the ThermostatDaySchedule struct represents an array of blocks. The purpose of this struct is to represent the schedule of one day a.k.a a series of blocks. Last but not least, the outermost struct, "ThermostatSchedule", maps each day of the week (Sunday - Saturday) to their respective daily schedules (ThermostatDaySchedule struct). This is the struct that is delivered to the user for getting and setting purposes.
 
+##### Thermostat Block Schedule Struct Fields Explanation
+
+- CoolSetting: The cool setting refers to the temperature at which the system begins cooling. In other words, if the room temperature surpasses this threshold, the cooling system is activated. The unit of temperature is Fahrenheit.
+- HeatSetting: The heat setting refers to the temperature at which the system begins heating. In other words, if the room temperature falls below this threshold, the heating system is activated. The unit of temperature is Fahrenheit.
+- System: This indicates what the system is currently doing. There are four possible settings (heat/cool/off/auto) which are pretty self explanatory. Heat and cool mean the systems heating or cooling the room. Auto means that the system will automatically heat or cool according to the room temperature and cool/heat thresholds.
+- Time: Time describes what time of day the particular block's settings are enacted (i.e. 6:00:AM). This time is in the RRule format (https://tools.ietf.org/html/rfc5545), and the rrule-go library is used to convert the given time into the designated format. The following section describes how time is formatted and defined in greater detail.
+
+##### A Deeper Dive into Time Format (RRule)
```

Comment:
Can you go into more detail on the translation of the RRule format into a string? I like the functional explanation so far!

Reply:
No problem! Just added an extra two paragraphs at the end of the "Deeper Dive into Time Format" paragraph. I wrote it so that it points out the implementation of the RRule function that performs the conversion.